### ISSUE TITLE
'2': 'Invalid Parameter - -resize\r\n',

### ISSUE THREAD

**Issue by LingboTang** [aheckmann/gm#528]
I'm running the resize.js example on windows 10 using node. And I got this error:

```
{ '0':
   { [Error: Command failed: Invalid Parameter - -resize
   ] code: 4, signal: null },
  '1': '',
  '2': 'Invalid Parameter - -resize\r\n',
  '3': 'convert "C:\\Users\\ltang\\Documents\\GitHub\\testMyGM\\imgs\\Balls.jpg" "-resize" "58x50%" "C:\\Users\\ltang\\Documents\\GitHub\\testMyGM\\imgs\\newImage.jpg"' }
```

And this is how I modified the resize example 

```
var gm = require('gm').subClass({
    imageMagick: true
});

var currentDir = __dirname + "\\imgs";

//var gm = require('gm');

console.log(currentDir);

gm(currentDir + "\\image.jpg").resize(353, 257, "!")
        .autoOrient()
        .write(currentDir+"\\newImage.jpg", 
        function (err) {
            if (!err) console.log(' hooray! '); console.log(err);
    });

```

Is there anything wrong with my syntax and how can I resolve it?

**Comment by gablabelle** [comment]
@LingboTang I have the same problem ... Did you find a solution ?

  
```
    var thumb1280 = task.target.replace('.jpg', '-1280.jpg');

    var resizeThePhotos = function(targetRelativePath) {

        var target = path.join(__dirname, '..', targetRelativePath);

        gm(target)
            .resize('1280')
            .write(thumb1280, function (err) {
                if (!err) {
                    //console.log(target + ': 1280 done');
                }
                else {
                    console.log(target, err);
                }
            });
    };
```


I get { [Error: Command failed: Invalid Parameter - -resize] code: 4, signal: null }

**Comment by LingboTang** [comment]
No I didn't fix the original, it must be some syntax problem. But I did some kind of hack, because I'm using gm to thumbnail images in S3 bucket. How I did to make it work is to include imageMagick:

```
var gm = require('gm').subClass({
    imageMagick: true
});
```

**Comment by gablabelle** [comment]
OK ... I reinstalled ImageMagick checking legacy tools got a different error ... so decided set it to false instead of true and it works now.

`var gm = require('gm').subClass({imageMagick: false});`

**Comment by LingboTang** [comment]
Ok, that's interesting, I don't know where exactly cause the problem. I think there should be some explanation about this. I'll keep this issue here.

**Comment by mattbajorek** [comment]
I had a similar issue and had solved the issue by making sure I installed the legacy utilities.

![legacyutilities](https://cloud.githubusercontent.com/assets/17235301/16716639/7eb5c91c-46d0-11e6-89dd-696ead835039.png)

**Comment by neal1991** [comment]
@mattbajorek  Hi, I have tried to install imgamagick. But I have not seen the `install legacy utilities`. Which version you have installed, I selected verison `ImageMagick-7.0.7-2-Q16-x64-static.exe`.

**Comment by aheckmann** [comment]
Fixed by #846. Released in v1.25.0.
